pkgs/development/python-modules/sphinx-markdown-parser/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, sphinx
+, markdown
+, CommonMark
+, recommonmark
+, pydash
+, pyyaml
+, unify
+, yapf
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx_markdown_parser";
+  version = "0.2.4";
+
+  # PyPi release does not include requirements.txt
+  src = fetchFromGitHub {
+    owner = "clayrisser";
+    repo = "sphinx-markdown-parser";
+    rev = "2fd54373770882d1fb544dc6524c581c82eedc9e";
```

Comment:
https://github.com/clayrisser/sphinx-markdown-parser/issues/34

Reply:
Thanks, however this seems to be sadly quite common behavior for python packages.

Reply:
Can you add a comment that this rev is the release one? Otherwise another reviewer might request that the version number ist changed to unstable-XXXX-XX-XX.

Reply:
Good idea. Done.